### PR TITLE
Update tutorial documentation

### PR DESCRIPTION
**Pull request by keharper**
## This PR is a:

- [ ] New topic
- [ ] Content update
- [ ] Content fix or rewrite
- [x] Bug fix or improvement

## Summary

This pull request updates the tutorial documentation. Several metadata parameters have been added and deleted since the last major revision of these topics.

### CODE REVIEW

File: guides/v2.1/contributor-guide/templates/tutorial-template-first.md [hunk 1]
```diff
@@ -1,35 +1,47 @@
...
 
-* When a user clicks on a tutorial title from the left navigation area, the Devdocs {% glossarytooltip a3c8f20f-b067-414e-9781-06378c193155 %}website{% endglossarytooltip %} replaces the navigation with a table of contents that is specific to the tutorial.
+* A tutorial replaces the navigation with a table of contents that is specific to the tutorial.
+* The bottom of the index page contains a **Begin Tutorial** button.
 * **Next** and **Previous** buttons are displayed at the bottom of each topic.
 
 ## Metadata parameters
 
-The following table lists the {% glossarytooltip 3f0f2ef1-ad38-41c6-bd1e-390daaa71d76 %}metadata{% endglossarytooltip %} parameters that are used in a tutorial.
+The following table lists the {% glossarytooltip 3f0f2ef1-ad38-41c6-bd1e-390daaa71d76 %}metadata{% endglossarytooltip %} parameters that are used on the index page of a tutorial.
```
Comment (jeff-matthews) [inline guides/v2.1/contributor-guide/templates/tutorial-template-first.md]: Consider rewriting in active voice:

> You must use the following metadata parameters on the tutorial index page.

File: guides/v2.1/contributor-guide/templates/tutorial-template-first.md [hunk 2]
```diff
@@ -1,35 +1,47 @@
...
-`subgroup:`| First page only. Places a link with the specified text to the {% glossarytooltip 31751771-8163-434b-88bc-c5f94d859fc3 %}sidebar{% endglossarytooltip %}
+`title:` | The title of the tutorial.
+`menu_title:` | On the index page of the tutorial, specify `Initial tasks`.
 `level3_subgroup:` | A group name that binds the tutorial topics together. This value must be specified on each topic in the tutorial.
-`title:` | The title of the tutorial. For all other topics in the tutorial, this value should be in the form _Step X. Perform this task_. (For example, `Step 1. Configure the Store`)
-`subtitle` | Displays the tutorial name. The value must be specified on each topic in the tutorial.
-`menu_title:` | On the first page of the tutorial, specify `Initial tasks`. Otherwise, leave this line blank.
-`menu_order:` | The sequence number of the topic. If the topic is an intro topic in a multi-step tutorial, set this 0. Otherwise set it to the step number.
-`ee_only:` | If set to `true`, graphics/cues indicating that the article applies to EE are displayed on devdocs.
+`menu_order:` | The sequence number of the topic. For the index page in a multi-step tutorial, set this 0.
```
Comment (jeff-matthews) [inline guides/v2.1/contributor-guide/templates/tutorial-template-first.md]: "...set this **to** 0."

File: guides/v2.1/contributor-guide/templates/tutorial-template-first.md [hunk 3]
```diff
@@ -1,35 +1,47 @@
...
+`return_to:` | Defines the tutorial's parent page. The parent page title is displayed in the left navigation above the tuturial steps. Do not specify a value for the `return_to:` parameter. Instead, specify values for these second-level parameters:<br/><br/>`title:` The title of the parent topic<br/><br/>`url:` The URL of the parent topic
+`ee_only:` | Optional. If set to `true`, graphics/cues indicating that the article applies to EE are displayed on devdocs.
```
Comment (jeff-matthews) [inline guides/v2.1/contributor-guide/templates/tutorial-template-first.md]: Replace obsolete reference to "EE" with variable `{{site.data.var.ee}}`.

Also, apply style attribute to table: `{:style="table-layout:auto;"}`

File: guides/v2.1/contributor-guide/templates/tutorial-template-first.md [hunk 4]
```diff
@@ -1,35 +1,47 @@
...
+
+A completed frontmatter section for an index page:
```
Comment (jeff-matthews) [inline guides/v2.1/contributor-guide/templates/tutorial-template-first.md]: Consider rewriting using a complete sentence:

> The following example shows a complete frontmatter section.

File: guides/v2.1/contributor-guide/templates/tutorial-template-last.md [hunk 5]
```diff
@@ -1,20 +1,50 @@
...
 level3_subgroup: sample-tutorial
+return_to:
+   title: Code Contributions
+   url: contributor-guide/contributing.html
+
+
 ---
 
-The `title` and `subtitle` parameters provide the top-level headings on this page. Add the content as you see fit.
+The last page of tutorial should acknowledge the reader has completed the tutorial and provide links to related information. Otherwise, add the content as you see fit.
```
Comment (jeff-matthews) [inline guides/v2.1/contributor-guide/templates/tutorial-template-last.md]: "The last page of **a** tutorial..."

File: guides/v2.1/contributor-guide/templates/tutorial-template-last.md [hunk 6]
```diff
@@ -1,20 +1,50 @@
...
+Parameter | Description
+--- | ---
+`layout:` | Must be `tutorial`.
+`title:` | The title of the topic. This value should be in the form _Step X. Perform this task_. (For example, `Step 1. Configure the Store`)
+`menu_title:`| Optional. Use this parameter only if you don't want the title in the left navigation to match the topic title.
+`subtitle` | The title of the tutorial. This value should match the `title` value in the index file.
+`level3_subgroup:` | A group name that binds the tutorial topics together. This value must be specified on each topic in the tutorial.
+`menu_order:` | The sequence number of the topic. For the index page in a multi-step tutorial, set this 0. Otherwise set it to the step number.
+`return_to:` | Defines the tutorial's parent page. The parent page title is displayed in the left navigation above the tuturial steps. Do not specify a value for the `return_to:` parameter. Instead, specify values for these second-level parameters:<br/><br/>`title:` The title of the parent topic<br/><br/>`url:` The URL of the parent topic
+`ee_only:` | Optional. If set to `true`, graphics/cues indicating that the article applies to EE are displayed on devdocs.
```
Comment (jeff-matthews) [inline guides/v2.1/contributor-guide/templates/tutorial-template-last.md]: Replace obsolete reference to "EE" with variable `{{site.data.var.ee}}`.

Also, apply style attribute to table: `{:style="table-layout:auto;"}`

File: guides/v2.1/contributor-guide/templates/tutorial-template-last.md [hunk 7]
```diff
@@ -1,20 +1,50 @@
...
+  title: REST tutorials
+  url: rest/tutorials/index.html
+menu_order: 9
+level3_subgroup: order-tutorial
+```
 
 ## Secondary heading
 
-Use a Head2 (##) as the highest-leveling heading in this topic.
+Use a Head2 (`## Heading`) as the highest-leveling heading in this topic.
```
Comment (jeff-matthews) [inline guides/v2.1/contributor-guide/templates/tutorial-template-last.md]: "Use a Head2 (`## Heading`) as the highest-level heading..."

File: guides/v2.1/contributor-guide/templates/tutorial-template-middle.md [hunk 8]
```diff
@@ -1,17 +1,46 @@
...
+   url: contributor-guide/contributing.html
+
 ---
 
-The `title` and `subtitle` parameters provide the top-level headings on this page. Add the content as you see fit.
+Add the content for each step as you see fit.
+
+## Metadata parameters
+
+The {% glossarytooltip 3f0f2ef1-ad38-41c6-bd1e-390daaa71d76 %}metadata{% endglossarytooltip %} parameters for a middle page of a tutorial are the same as the index page, except middle pages also use the `subtitle` parameter.
```
Comment (jeff-matthews) [inline guides/v2.1/contributor-guide/templates/tutorial-template-middle.md]: Consider rewrite:

> The metadata parameters for all pages between the first and last page of a tutorial...

File: guides/v2.1/contributor-guide/templates/tutorial-template-middle.md [hunk 9]
```diff
@@ -1,17 +1,46 @@
...
+  title: REST tutorials
+  url: rest/tutorials/index.html
+menu_order: 1
+level3_subgroup: order-tutorial
+```
 
 ## Secondary heading
 
-Use a Head2 (##) as the highest-leveling heading in this topic.
+Use a Head2 (`## Heading`) as the highest-leveling heading in this topic.
```
Comment (jeff-matthews) [inline guides/v2.1/contributor-guide/templates/tutorial-template-middle.md]: "Use a Head2 (## Heading) as the highest-level heading..."

File: guides/v2.1/contributor-guide/templates/tutorial-template-middle.md [hunk 10]
```diff
@@ -1,17 +1,46 @@
...
+Parameter | Description
+--- | ---
+`layout:` | Must be `tutorial`.
+`title:` | The title of the topic. This value should be in the form _Step X. Perform this task_. (For example, `Step 1. Configure the Store`)
+`menu_title:`| Optional. Use this parameter only if you don't want the title in the left navigation to match the topic title.
+`subtitle` | The title of the tutorial. This value should match the `title` value in the index file.
+`level3_subgroup:` | A group name that binds the tutorial topics together. This value must be specified on each topic in the tutorial.
+`menu_order:` | The sequence number of the topic. For the index page in a multi-step tutorial, set this 0. Otherwise set it to the step number.
+`return_to:` | Defines the tutorial's parent page. The parent page title is displayed in the left navigation above the tuturial steps. Do not specify a value for the `return_to:` parameter. Instead, specify values for these second-level parameters:<br/><br/>`title:` The title of the parent topic<br/><br/>`url:` The URL of the parent topic
+`ee_only:` | Optional. If set to `true`, graphics/cues indicating that the article applies to EE are displayed on devdocs.
```
Comment (jeff-matthews) [inline guides/v2.1/contributor-guide/templates/tutorial-template-middle.md]: Replace obsolete reference to "EE" with variable `{{site.data.var.ee}}`.

Also, apply style attribute to table: `{:style="table-layout:auto;"}`

**Comment by keharper** [comment]
All review comments incorporated.

**Pull request merged**